Catalog/Examples/UI/ToggleButton/TogglePresenter.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  TogglePresenter.swift
+//  Catalog
+//
+//  Created by Domagoj Hustnjak on 4/4/20.
+//  Copyright © 2020 Infinum. All rights reserved.
+//
+
+import RxSwift
+import RxCocoa
+import RxRelay
+
+final class TogglePresenter {
+
+    // MARK: - Private properties -
+
+    private unowned let view: ToggleViewInterface
+    private let wireframe: ToggleWireframeInterface
+    private let interactor: ToggleInteractorInterface
+
+    // MARK: - Lifecycle -
+
+    init(wireframe: ToggleWireframeInterface, view: ToggleViewInterface, interactor: ToggleInteractorInterface) {
+        self.wireframe = wireframe
+        self.view = view
+        self.interactor = interactor
+    }
+}
+
+// MARK: - Extensions -
+
+extension TogglePresenter: TogglePresenterInterface {
+
+    func configure(with output: Toggle.ViewOutput) -> Toggle.ViewInput {
+        let relay = PublishRelay<ToggleState>()
```

Comment:
I'd rename this to something like `progressRelay` to make its purpose easier to understand 🙂